pipelines/optimus/Optimus.wdl

```diff
@@ -208,33 +208,42 @@ workflow Optimus {
       col_index = MergeCountFiles.col_index
   }
 
-  if (output_zarr) {
-    call ZarrUtils.OptimusZarrConversion{
+  call ZarrUtils.OptimusZarrConversion{
+    input:
+      sample_id=sample_id,
+      annotation_file=annotations_gtf,
```

Comment:
There should be spaces surrounding the `=`.

Reply:
done